lesson5.html

```diff
@@ -0,0 +1,42 @@
+<html>
+    <head>
+        <title>Mouse leave and over event</title>
+        <link rel="stylesheet" type="text/css" href="style5.css">
+    </head>
+    <body>
+        <div id ="newBtn"></div>
+        <div class="btn-postn"><button id ="awsome">Just-Awsome</button></div>
+        
+        <script>
+            let awsmBttnCmp = document.getElementById("awsome");
+            let newCmp = document.getElementById("newBtn");
+            const label = "Im Awsome" ;
+             function onClick(event){
+                 console.log("Hey!! Be positive");
+             }
+             function mouseOverHandler (event){
+                 console.log("You are here");
+                 awsmBttnCmp.innerText = "Be Awsm" ;
+                 awsmBttnCmp.classList.add("original-color");
```

Comment:
you can unite color and shape css in one class, then you will have to add/remove only that one class instead of 2 separate.